src/logger.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as fs from "fs";
+import * as vscode from "vscode";
+import TelemetryReporter from "vscode-extension-telemetry";
+
+export enum Type {
```

Comment:
Actually this file is deprecated by the debugger. Now we're using the telemetry wrapper to send logger.

Reply:
Will move `Type` to utility.ts and logger.ts can be removed from this PR.